grape_log_formatter.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "lograge"
   spec.add_dependency "grape_logging", "~> 1.8.0"
```

Comment:
How about we add dependency here so you don't have to check the method is defined or not?

Reply:
I'd rather not have the dependency. That way if we swap to open tracing, we don't have to change this before we can do it.